lib/registrant.rb

```diff
@@ -0,0 +1,31 @@
+class Registrant
+    attr_reader :age,
+                :license_data,
+                :name,
+                :permit
+
+    def initialize(name, age, permit = false)
+        @name = name
+        @age = age
+        @license_data = {
+            written: false,
+            license: false,
+            renewed: false
+            }
+        @permit = permit
+               
+    end
+
+    def permit?
+        @permit == @permit
+    end
+
```

Comment:
This will always be true since youre checking if one thing is equal to itself ... which will always be true. You can just return `@permit` because that should store either true or false.

```ruby 
def permit? 
 @permit 
end 
```